packages/template-retail-react-app/app/utils/url.js

```diff
@@ -1,13 +1,14 @@
 /*
```

Comment:
The functions in this file is to process the given url and return a new one while `resolveSiteFromUrl` and `resolveLocaleFromUrl` is the other way around which it looks into the given url, extract and return data. I don't think they belong here 🤔